src/framework/uicomponents/view/popupview.h

```diff
@@ -24,13 +24,13 @@
 #define MU_UICOMPONENTS_POPUPVIEW_H
 
 #include <QQuickItem>
-#include <QQuickView>
 #include <QQmlParserStatus>
 
 #include "modularity/ioc.h"
 #include "ui/imainwindow.h"
 
 namespace mu::uicomponents {
+class MyQuickView;

```

Comment:
Let's rename it into "QuickViewWrapper or Adapter or Handler"

Reply:
Do we need to complicate?
I usually use the prefix `my` if the scope is only for a given object.

Reply:
Renamed and restructured